cardinal/world.go

```diff
@@ -395,6 +395,9 @@ func (w *World) ShutDown() error {
 	if w.evmServer != nil {
 		w.evmServer.Shutdown()
 	}
+	if w.server != nil {
+		w.server.Shutdown()
+	}
```

Comment:
i think this change lets us just do away with game manager? correct me if im wrong @pyrofolium @jerargus

Reply:
When is engine.Shutdown called? that used to be a responsibility of the game manager. I think engine.Shutdown belongs right here. i.e. the server is NOT responsible for shutting down the engine.

Reply:
got it, adding